scripts/deploy-transport-controller.sh

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-DEFAULT_TRANSPORT_CONTROLLER_IMAGE=ghcr.io/kubestellar/ocm-transport-plugin/transport-controller:0.1.11
+DEFAULT_TRANSPORT_CONTROLLER_IMAGE=ghcr.io/kubestellar/kubestellar/ocm-transport-controller:0.1.11
```

Comment:
I would expect images in kubestellar/kubestellar to be tagged with ks/ks release numbers.

What is the purpose of this script? The answer should be documented. I am wondering if this script should now be using a released image or building an image from the local source.

Reply:
Currently, this variable's setting does not matter. This script is always invoked with a third positional argument, so the default image does not matter. We currently have no documentation telling users that that they can call this script with only positional two arguments.

Perhaps we used to have such user instructions. Perhaps we will again in the future.

In the meantime, I think the simplest change in the definition of this variable is no change.

The repo merge campaign will include merging the release procedures and documents. Not necessarily in this PR.

Reply:
@MikeSpreitzer you're right. atm this variable is never used.  
maybe the best thing to do would be just to remove it. 
in case it will be needed in the future we can always re-add it.